### ISSUE TITLE
mne anonymize command

### ISSUE THREAD

**Issue by agramfort** [mne-tools/mne-python#4982]
we should have the command equivalent of

```
$ mne_anonymize
usage: mne_anonymize [options]
Remove subject information from a fif file.
	--his             remove the HIS ID tag as well
	--file name       the file to modify.
	--help            print this info.
	--version         print version info.
```

**Comment by dokato** [comment]
I'm happy to take it, if no one started.

**Comment by agramfort** [comment]
go for it.

it would be nice to allow be able to operate inplace too but fine to do it
in a subsequent PR.

**Comment by dokato** [comment]
okay, but please help me understand `--his` parameter.
Since in `mne.io.meas_info.anonymize_info` we remove whole `subject_info` key so `his_id` as well by default. Should I modify `anonymize_info` function to preserve this depending on some `his` flag?

**Comment by agramfort** [comment]
removing it all the time is good. Forget the --his param.

**Comment by drammock** [comment]
this was closed in #6892
